app/controller/settings.js

```diff
@@ -0,0 +1,12 @@
+var exports = module.exports = {};
+	
+
+exports.config = function (configData, req) {
```

Comment:
What are you doing? Config needs to be an object with methods on it for update and a data obj as well

var config = {};
config.data = require('config.json).config;
config.update = function(post) {
    1. Build new config JSON
    2. Save as config.json
}

This is basic level js stuff!